target/linux/generic/backport-5.15/733-v6.3-18-net-ethernet-mtk_eth_soc-add-support-for-MT7981.patch

```diff
@@ -1,3 +1,13 @@
+From 7472476aef263bbf49b19d084875caddb7fe8f2a Mon Sep 17 00:00:00 2001
+From: Felix Fietkau <nbd@nbd.name>
+Date: Wed, 29 Mar 2023 11:38:30 +0200
+Subject: [PATCH] kernel: fix mtk_eth_soc throughput regressions on gigabit PHY
+ ports
+
+Fixes issues on MT7621 and MT7623 switch ports, and PHY ports on all SoC
+
+Signed-off-by: Felix Fietkau <nbd@nbd.name>
```

Comment:
This patch has a different commit message: 
https://git.kernel.org/pub/scm/linux/kernel/git/netdev/net-next.git/commit/?id=f5d43ddd334b7c32fcaed9ba46afbd85cb467f1f

Reply:
I couldn't find the original, so I took it from the openwrt commit message.

I've pulled that patch instead and replaced it.

Would have been easier if we would 'get it right' from the get-go :)

Thanks @hauke

Reply:
Yes it would have been better if the commit description would have been added directly before the patch was added. 
When a patch is in the backports directory it should be in linux-next too, I was just looking there for the changes.